tox.ini

```
@@ -2,9 +2,9 @@
 usedevelop = True
 
 envlist =
-    py{36,37,38,39}-dj{22,31}-wagtail{211,212,213}
-    py{36,37,38,39}-dj31-wagtail214
-    py{38,39}-dj{32}-wagtail{213,214}
+    py{37,38,39}-dj{22,31}-wagtail{211,213}
+    py{37,38,39}-dj31-wagtail214
+    py{37,38,39,310}-dj{32}-wagtail{214,215}
```

According to [Wagtail’s compatibility table](https://docs.wagtail.io/en/stable/releases/upgrading.html#compatible-django-python-versions), only Wagtail 2.15 and up support Python 3.10, so I believe this should be:

```suggestion
    py{37,38,39}-dj{32}-wagtail{214,215}
    py{310}-dj{32}-wagtail{215}
```

---

Happy to go with the official charts. Thank you @thibaudcolas.
unofficially, it worked well before 2.15 according to the tests :D